userApi/src/main/java/cs4337/group9/mediumwebsite/enums/Role.java

```diff
@@ -1,5 +1,7 @@
 package cs4337.group9.mediumwebsite.enums;
 
 public enum Role {
-    ADMIN, AUTHOR
-}
+    ADMIN,   // Admin role
+    AUTHOR,  // Author role
+    USER     // User role (added this)
```

Comment:
I dont think we need a new role here for USER, as in our case an AUTHOR is a user, we could probably refactor author to user to make it more clear in the future